recipe_scrapers/cookwell.py

```diff
@@ -1,7 +1,29 @@
 from ._abstract import AbstractScraper
+import re
 
 
 class CookWell(AbstractScraper):
     @classmethod
     def host(cls):
         return "cookwell.com"
+
+    def nutrients(self):
+        nutrition = None
+        nutrition_pattern = re.compile(
+            r"{\\\"calories\\\":(?P<calories>\d+),\\\"carbohydrates\\\":(?P<carbohydrates>\d+),\\\"fat\\\":(?P<fat>\d+),\\\"protein\\\":(?P<protein>\d+)}"
+        )
+
+        nutrition_script = self.soup.find("script", string=nutrition_pattern)
+        if nutrition_script:
+            nutrition_info = nutrition_pattern.search(
+                nutrition_script.string
+            ).groupdict()
+            if nutrition_info:
+                nutrition = {
+                    "calories": f'{nutrition_info["calories"]} calories',
```

Comment:
```suggestion
                    "calories": nutrition_info["calories"],
```

According to the [NutritionInformation Schema](schema.org/NutritionInformation) spec we should be expecting the number of calories as the result here as opposed to some other fields which expect the number of grams (or other unit of measurement).

Reply:
Gotcha, I did it that way to mirror the value captured from the site's recipe metadata, which doesn't exactly follow the spec.
```
from urllib.request import urlopen
from recipe_scrapers import scrape_html
url = "https://cookwell.com/recipe/crispy-oven-fries"
html = urlopen(url).read().decode("utf-8")
scraper = scrape_html(html,url)
scraper.schema.nutrients()
```
> {'calories': '970 calories'}

But I'm fine with the change, adding it to the revision

Reply:
This is just a change we implement on this end so that the output can be expected the same throughout all coverage. See #1304 & #1294 for two other instances of something similar.